src/containers/login/Login.tsx

```diff
@@ -54,17 +59,47 @@ const Footer: typeof Paragraph = styled(Paragraph)<ParagraphProps>`
   margin-bottom: 10px;
 `;
 
+const LoginAlert = styled(Alert)`
+  width: 90%;
+  margin-top: -60px;
+  margin-bottom: 20px;
+`;
+
+const MobileLoginAlert = styled(Alert)`
+  width: 90%;
+  margin-bottom: 20px;
+`;
+
+const loginErrorMessage = 'The username or email you entered was incorrect.';
+const greetingHeader = 'Welcome Back!';
+const greetingBody =
+  'Dreamcatcher kogi taiyaki keytar. Swag typewriter craft beer\n' +
+  'cronut pok pok gentrify flannel salvia deep v pork belly\n' +
+  'pitchfork. Swag fashion axe fam. Occupy biodiesel jean shorts\n' +
+  'affogato PBR&B freegan bushwick vegan four loko pickled.';
+
 type LoginProps = UserAuthenticationReducerState;
 
 const Login: React.FC<LoginProps> = ({ tokens }) => {
   const { windowType, width } = useWindowDimensions();
   const dispatch = useDispatch();
-  const onFinish = (values: any) => {
+  const history = useHistory();
+  const privilegeLevel = useSelector((state: C4CState) =>
+    getPrivilegeLevel(tokens),
+  );
```

Comment:
```suggestion
  const privilegeLevel = useSelector((state: C4CState) =>
    getPrivilegeLevel(state.authenticationState.tokens),
  );
```

This selector should reference the state that it is provided rather than the prop. This could have effects on the Redux lifecycle/rules of hooks. Small issue though, probably fine either way.